hypervisor/arch/x86/guest/vcpu.c

```diff
@@ -653,6 +655,8 @@ int32_t run_vcpu(struct acrn_vcpu *vcpu)
 	/* Obtain current VCPU instruction length */
 	vcpu->arch.inst_len = exec_vmread32(VMX_EXIT_INSTR_LEN);
 
+	vcpu->arch.proc_vm_exec_ctrls = exec_vmread32(VMX_PROC_VM_EXEC_CONTROLS);
```

Comment:
If we cached it, we never read it from VMCS field.

Reply:
Now, It can be changed by nmi_window and mtf. So we need to cache it when vmexit and write back before vmentry.

Reply:
Not addressed.